README.md

```diff
@@ -607,4 +618,4 @@ Thanks to,
 
 ---
 
-This software is in no way derived from or based on Jackson Yee's abandoned 2006 "pyembroidery" project. The name was simply taken from libEmbroidery and written in python and thus a portmanteau of those. I was unaware of the project until after the all the principal work on this project was complete. I apologize for any confusion this may cause.
+This software is in no way derived from or based on Jackson Yee's abandoned 2006 "pystitch" project. The name was simply taken from libEmbroidery and written in python and thus a portmanteau of those. I was unaware of the project until after the all the principal work on this project was complete. I apologize for any confusion this may cause.

```

Comment:
```suggestion
This software is in no way derived from or based on Jackson Yee's abandoned 2006 "pyembroidery" project. The name "pyembroidery" for the project from which this was forked was simply taken from libEmbroidery and written in python and thus a portmanteau of those. Awareness of Jackson Yee's project was only obtained after all the principal work on this project was complete - apologies for any confusion this may cause.

```
Jackson Yee's project was called pyembroidery not pystitch.